packages/jest-cli/src/runJest.js

```diff
@@ -90,12 +89,11 @@ const processResults = (runResults, options) => {
   }
   if (isJSON) {
     if (outputFile) {
-      const filePath = path.resolve(process.cwd(), outputFile);
```

Comment:
This changes where the JSON is written if `outputFile` is relative, right?
Do we need a test / changelog entry for this change?

Reply:
Yeah, but I think it makes sense to output a runs result file to `rootDir` more than `cwd`. Maybe it should have a test? @thymikee thoughts on that

Reply:
`rootDir` makes more sense, but yea it's a breaking change and we should communicate it. Maybe send a separate PR with only this?

Reply:
done (well, not the separate PR for moving it, but reverted to `cwd`)